clang_3.9/sources.list

```diff
@@ -0,0 +1,18 @@
+deb http://old-releases.ubuntu.com/ubuntu/ artful main restricted
```

Comment:
is this file different from `clang_3.9-x86/sources.list`?

Reply:
only a new line is different, but the content is the same. I checked with vimdiff

Reply:
okay, may we reduce copy-paste somehow? e.g. move file to some shared location, or just use it in base image once?

Reply:
I think the second options is better, since we can't use relative path when copying (docker feature).

Reply:
I cant use the second options ... docker-compose uses each image folder as a context, which means it won't the sources.list file. I could use symbolic link WDYT ?

Reply:
that's unfortunate, okay, let's leave it for now as is